### ISSUE TITLE
Issue with homing xy 

### ISSUE THREAD

**Issue by Alemontfer1992** [re3Dprinting/touchscreen#53]

**Comment by jim-thompson** [comment]
I believe if you follow home XY with a home Z, it should take care of that.

**Comment by jim-thompson** [comment]
Is this different behavior than what you get if you home XY using Viki?

**Comment by Alemontfer1992** [comment]
Yup. When you home XY using Viki, I believe only XY move and the Z doesn't lower at all. I just found it weird because I home Z first, then XY and then I had to home Z again. This way, it's sort of forcing you to home all but in a 2 step process.

**Comment by jim-thompson** [comment]
OK, I'll check it out. Thanks for the feedback.

**Comment by chief-hacker** [comment]
> Yup. When you home XY using Viki, I believe only XY move and the Z doesn't lower at all. I just found it weird because I home Z first, then XY and then I had to home Z again. This way, it's sort of forcing you to home all but in a 2 step process.

Possible solution - Have a "Home All" button that will allow you to home everything with one button push.

**Comment by jim-thompson** [comment]
We have a Home All button, next to the Motor Off button. It's not labeled.
